### ISSUE TITLE
(Mis)Using the importer for IKEA Kitchen Planner

### ISSUE THREAD

**Issue by jnettels** [eliemichel/MapsModelsImporter#326]
- Your GPU name GeForce GTX 1070
 - Version of this add-on: 0.7.0
 - Version of RenderDoc: 1.31
 - Version of Blender: 4.1.1

Hi! The intended purpose of RenderDoc + this Add-On works fine for me. I can import 3D Google Maps models into Blender (very nice, thank you very much!).

A reddit thread suggested trying this to capture 3D models from the IKEA Kitchen Planner (which has no 3D export function). Having only tried this for the first time for a couple of minutes, of course I have no idea if this is even remotely feasible. If someone is looking for a challenge and wants to have a look at it, here is my capture:

https://1drv.ms/u/s!AvuqseHmWKh0icoZM1Bf0-ILUMshJg?e=lW3dkI

Currently the following is reported:
```
Trying scraping strategy #8 (from draw call #2145)...
Error: Could not find the beginning of the relevant 3D draw calls
```

Cheers!

**Comment by yemsb** [comment]
Hey, did you ever make any progress on this @jnettels? I would also be interested in getting the 3D models of a planned IKEA kitchen. 

Thanks!

**Comment by jnettels** [comment]
@yemsb No, I did not put any further effort into it. I needed a representation of a kitchen, because I wanted to plan and visualize renovation ideas for my house in Blender. I ended up using the measurements from the kitchen planner to rebuild a simpler representation in Blender.

However something helpful, that maybe not everyone knows: 
You can download 3D models of many individual products.
- Go to e.g. https://www.ikea.com/de/de/p/bagganaes-griff-schwarz-80338413/
- In Firefox, press F12 for developer menu, then go to "Network-Analysis" ("Netzwerkanalyse")
- Hit the button "3D-View" on the IKEA page. The developer menu must be opened before loading the 3D view for the first time, because of caching I think.
- Use the filter in Network-Analysis to filter for ".glb" files.
- A file like this is then found: https://web-api.ikea.com/dimma/assets/1.2/80338413/PS01_S01_NV01/rqp3/glb_draco/fe016af6856f413da9ce110032d950f8-80338413_PS01_S01_NV01_RQP3_3.0.glb
- Right-click on the file name to "Open in new Tab"
- You just downloaded a "GLB/GLTF" file, that can directly be imported into Blender. It is the same object as viewed in the webbrowser, fully textured and to scale. This way you can still decorate the kitchen with IKEA products, place door handles and other stuff that is available as individual products.

I also used this method for one of the other IKEA planners (PAX, I think). I was able to download each individual element represented in the planner. Afterwards I still had to assemble all the pieces in Blender... just like real life, I guess.
